Docs/Book/Install.md

```diff
@@ -33,7 +33,40 @@ Windows users will use a slightly different command:
 
 ### How to build from source with Conda
 
-For more details on building from source with Conda, see the [conda-rdkit repository](https://github.com/rdkit/conda-rdkit)
+For more details on building from source with Conda, see the [conda-rdkit repository](https://github.com/rdkit/conda-rdkit).
+
+#### macOS 10.12 (Sierra): Python 3 environment
+
+The following commands will create a development environment for macOS Sierra and Python 3. Download
+Miniconda3-latest-MacOSX-x86_64.sh from [Conda](http://conda.pydata.org/miniconda.html) and run these
+following commands:
+   
+	bash Miniconda3-latest-MacOSX-x86_64.sh
+	conda install numpy scipy matplotlib
+	conda install cmake
+	conda install -c  https://conda.anaconda.org/rdkit  boost
+	conda install -c  https://conda.anaconda.org/rdkit  cairocffi
+	conda install pillow
+	conda install anaconda
+	conda install --channel https://conda.anaconda.org/rwest eigen3
```

Comment:
I have been installing eigen from conda-forge.
Given that this is somewhat more "official" than rwest's repo, I would suggest using that:
`conda install -c conda-forge eigen`